docs/team/alanlowzies.md

```diff
@@ -1,6 +1,103 @@
+should take note of to avoid issues in the application.</span>
+<span class="box info">🧾  This is an information box. Contains information that may be useful for
+the users.</span>
+
+### Contributions to the User Guide (UG)
+- Wrote the guide on [creating a new workout](../UserGuide.md#create-a-workout-workout-new).
+- Wrote the informational section on [WerkIt!'s local storage structure](../UserGuide.md#werkits-local-storage-information).
+- Wrote the [Quick Start Guide](../UserGuide.md#quick-start-guide) section, which guides the user on the necessary
+prerequisites to download and the steps to get it running on the user's computer.
+  - Includes a list of recommended OSes, their versions, and the terminals to use for each OS.
+- Wrote the [frequently asked questions](../UserGuide.md#frequently-asked-questions-faq) section.
+
+### Contributions to the Developer Guide (DG)
+- Wrote the following contributions to the ['Implementation'](../DeveloperGuide.md#implementation) section:
+    - How WerkIt! repeatedly waits and receives user inputs ([link to part](../DeveloperGuide.md#getting-user-input-continuously)).
+    - How WerkIt! parses user inputs and determines the appropriate procedures to take 
+      ([link to part](../DeveloperGuide.md#parsing-user-input-and-getting-the-right-command)).
+    - How WerkIt! processes and creates a new workout when requested by the user
+      ([link to part](../DeveloperGuide.md#create-new-workout)).
+      - Includes sequence diagrams to visualise the process.
+- Wrote the instructions on 
+[how to set up the development environment](../DeveloperGuide.md#setting-up-your-development-environment).
+- Wrote part of the [glossary](../DeveloperGuide.md#glossary)
+- Wrote the [table of contents](../DeveloperGuide.md#table-of-contents).
+
+### Contributions to Team-Based Tasks
+- Set up the team's organisation and repository on GitHub
+- Set up and maintained the team's Google Drive folder, which includes the team's project notebook
+and other documents for the co-requsuite module CS2101 Effective Communication for Computing Professionals.
+- Helped to maintain the team's GitHub repository issue tracker.
+
+### Review/Mentoring Contributions
+- Reviewed fellow team members' pull requests (PRs), including (but no limited to):
+  - [PR #71 - Add logging and assertion for update workout](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/71#pullrequestreview-910244404)
+  - [PR #97 - Add Create Plan Feature](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/97#pullrequestreview-914977219)
+  - [PR #123 - Branch search workout](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/123#pullrequestreview-919593971)
+  - [PR #134 - Improve delete workout](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/134#pullrequestreview-920276634)
+  - [PR #218 - Branch file bug](https://github.com/AY2122S2-CS2113T-T09-2/tp/pull/218#pullrequestreview-929510891)
```

Comment:
felt like you reviewed more hmmm... perhaps you can't find or still editing x)

Reply:
Haha I did review more uh, but I didn't want to list them all, as I thought it could be unnecessarily too much 😬 hence I just picked 5 PRs that I felt I gave substantial review comments :)